Unable to compile even the blink.ino sketch

The bellow error appeared to my Arduino IDE ( Running in Windows 10 ) :
In file included from E:\ARDUINO_ESP_DEV\arduino-IDE\hardware\esp8266com\esp8266\cores\esp8266\core_esp8266_main.cpp:35:0:

C:\Users\kranidis\AppData\Local\Temp\arduino_build_25694/core/core_version.h:1:33: error: unable to find numeric literal operator 'operator"" xffffffff'

 #define ARDUINO_ESP8266_GIT_VER 0x0xffffffff

                                 ^

E:\ARDUINO_ESP_DEV\arduino-IDE\hardware\esp8266com\esp8266\cores\esp8266\core_esp8266_main.cpp:65:78: note: in expansion of macro 'ARDUINO_ESP8266_GIT_VER'

 extern const uint32_t __attribute__((section(".ver_number"))) core_version = ARDUINO_ESP8266_GIT_VER;

                                                                              ^
exit status 1
Error compiling for board Generic ESP8266 Module.

I have ESP8266 updated from GIt Master branch

Please keep an eye what is wrong and I am here to supply you any further information you may ask.

Thanks and Best Regards,
Mike Kranidis

--- a/tools/makecorever.py
+++ b/tools/makecorever.py
@@ -22,7 +22,7 @@ import os
 import subprocess
 
 
-def generate(path, platform_path, git_ver="0xffffffff", git_desc="unspecified"):
+def generate(path, platform_path, git_ver="ffffffff", git_desc="unspecified"):
     def git(*args):
         cmd = ["git", "-C", platform_path]
         cmd.extend(args)

